### PR TITLE
More array langs

### PR DESCRIPTION
**Pull request by hoosierEE**
Added a few more things; fixed Goal creator's first name and added John Earnest (creator of Decker, Lil, oK).

@mlochbaum also suggested adding https://aplwiki.com/wiki/Timeline_of_array_languages as a resource for array languages so let me know if there's a preferred spot I can add it.

### CODE REVIEW

File: concepts/goal.scroll [hunk 1]
```diff
@@ -3,7 +3,7 @@ import ../code/conceptPage.scroll
 id goal
 name Goal
 appeared 2022
-creators Marina Fernández de Retana
+creators Yon Fernández de Retana
```
Comment (breck7) [inline concepts/goal.scroll:6]: Ok, I changed `frundis.scroll` to "Yon" as well. Let me know if that was a mistake. Thanks!

**Comment by breck7** [comment]
Awesome! Thanks!

We could add that link to "https://pldb.io/lists/resources.html", next to lambda the ultimate perhaps.

Could also add it as a reference to any/all of the array language pages it references.

File: concepts/decker.scroll [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+id decker
+name Decker
+appeared 2022
+creators John Earnest
+tags pl
+website https://beyondloom.com/decker/
+conceptDescription Decker is a multimedia platform for creating and sharing interactive documents, with sound, images, hypertext, and scripted behavior.
+
+influencedBy HyperCard Twine Bitsy
```
[the remark this answers is not in the document]
Comment (hoosierEE) [reply concepts/decker.scroll:11]: [Twine](https://twinery.org/) is for interactive stories and [Bitsy](https://bitsy.org/) is for stories and games.

File: concepts/goal.scroll (hunk 1 above)
Comment (hoosierEE) [reply concepts/goal.scroll:6]: Yep, it was brought to my attention by Yon (I didn't know either) on an array language discord.

File: concepts/decker.scroll (hunk 2 above)
[the remark this answers is not in the document]
Comment (breck7) [reply concepts/decker.scroll:11]: Thanks! Very interesting. I hadn't seen those before. Added.

[the remark this answers is not in the document]
Comment (hoosierEE) [reply concepts/decker.scroll:11]: They were new to me too. Rabbit holes everywhere... 😁

[the remark this answers is not in the document]
Comment (breck7) [reply concepts/decker.scroll:11]: "Rabbit holes everywhere". Love that

**Pull request merged**